zkevm-circuits/src/evm_circuit/execution/logs.rs

```diff
@@ -47,12 +47,12 @@ impl<F: Field> ExecutionGadget<F> for LogGadget<F> {
     const EXECUTION_STATE: ExecutionState = ExecutionState::LOG;
 
     fn configure(cb: &mut EVMConstraintBuilder<F>) -> Self {
-        let mstart = cb.query_cell_phase2();
-        let msize = cb.query_word_rlc();
+        let mstart = cb.query_word32();
+        let msize = cb.query_word32();
 
         // Pop mstart_address, msize from stack
-        cb.stack_pop(mstart.expr());
-        cb.stack_pop(msize.expr());
+        cb.stack_pop(mstart.expr().to_word());
```

Comment:
This part looks strange to me. It queries a `word32` cell, convert to `expr` and then convert to `word`. If we don't declare the local var, `mstart`, then this line would look like 
```
cb.stack_pop(cb.query_word32().expr().to_word());
```
Is it possible to do like
```
cb.stack_pop(mstart.to_word());
```